### PR TITLE
[QC-1227] Get beam type from PDP parameter

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Review by justonedev1 — COMMENTED** [review]
Are the string values from `pdp_beam_type` defined somewhere? I saw them mentioned by Andrea but without any link

**Comment by Barthelemy** [comment]
short answer: I don't know. 
@aferrero2707 do you know where they are defined ? 
or perhaps @ktf knows ?

**Comment by ktf** [comment]
No idea. @shahor02 maybe?

**Comment by shahor02** [comment]
`pdp_beam_type` are defined in the ECS interface and converted by the ODC to BEAMTYPE env.var, but I don't know its source code.

**Comment by Barthelemy** [comment]
Ok, thank you for the info. I conclude that the format of these values is defined somewhere in ODC.

File: Framework/src/runnerUtils.cxx [hunk 1]
```diff
@@ -91,6 +91,20 @@ std::string computeStringActivityField(framework::ServiceRegistryRef services, c
...
+{
+  // convert the beam type received from pdp into the format we use in flp/ecs
+  if (pdpBeamType == "pp") {
+    return "PROTON-PROTON";
+  } else if (pdpBeamType == "PbPb") {
+    return "Pb-Pb";
+  } else if (pdpBeamType == "pPb") {
+    return "Pb-PROTON";
+  } else {
+    return "";
```
Comment (justonedev1) [inline Framework/src/runnerUtils.cxx:104]: Maybe add some warning, that we met value that we did not expect?

Comment (Barthelemy) [reply Framework/src/runnerUtils.cxx:104]: yep

Comment (Barthelemy) [reply Framework/src/runnerUtils.cxx]: done, I let you have a look

**Review by justonedev1 — APPROVED** [review]

**Pull request merged**